ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -280,6 +283,19 @@ public void setHtml(WebView view, @Nullable String html) {
     }
   }
 
+  @ReactProp(name = "headers")
```

Comment:
I don't think adding a third prop that calls loadUrl is a good idea, there's actually to TODO comment in `setUrl` to do something about `setUrl` and `setHtml`. It could be a good time to fix that.

We could do something like a `source` prop that takes an object like 
```
{
  html: string,
  url: string,
  headers: Map<string, string>,
}
```
and then if `html` is defined, use it, else use `url` and if `headers` are defined, add them. That way changing between html, url and url with headers won't the make webview load the wrong thing because of the order props were set.